.github/workflows/push.yaml

```diff
@@ -53,3 +53,43 @@ jobs:
         with:
           name: redskyctl_linux_amd64
           path: dist/redskyctl-linux-amd64.tar.gz
+      - name: KinD (Kubernetes in Docker) Action
+        uses: engineerd/setup-kind@v0.3.0
+      - name: commands
+        run: |
+          kubectl get nodes -o wide
+          kubectl get node kind-control-plane -o yaml
+          kubectl get pods -n kube-system
+      - name: redsky quickstart
+        run: |
+          echo "Install redskyctl"
+          curl -sLo - https://github.com/redskyops/redskyops-controller/releases/download/v1.5.0/redskyctl-linux-amd64.tar.gz | \
+            tar -zxf - && \
+            chmod 755 ./redskyctl
+          echo "Init redskyops"
+          ./redskyctl init
+          echo "Install kustomize"
+          curl -s "https://raw.githubusercontent.com/\
+          kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | \
+            bash
```

Comment:
I wonder if the `kubectl -k` option is enough: we don't actually do much with these examples.

Reply:
No go with `kubectl -k` unfortunately. Looks like Kubectl bundles kustomize v2 and we need v3 to work with our kustomize files. :(